client.go

```diff
@@ -548,6 +548,82 @@ func (c *Client) SendInteractionResponse(ctx context.Context, interaction *Inter
 	return err
 }
 
+func (c *Client) CreateGlobalCommand(ctx context.Context, applicationID Snowflake, command *ApplicationCommand) error {
```

Comment:
Given that we only want to run these commands once (and not every time the server starts up) should this belong to a separate object altogether?

Reply:
I would simply use https://discord.com/developers/docs/interactions/application-commands#get-global-application-commands to figure what commands are already registered, before calling CreateGlobalCommand.

As of now, I simply think of the disgord api to make api calls available to you. Not so much dependent on the state of your bot for when you can call what. 

There could definetly be a helper method that fetches registered commands, diff and only registered new ones. But I think I want to wait until interactions are fully implemented.